src/components/orders-table/orders-data-table.component.tsx

```diff
@@ -128,7 +130,11 @@ const OrdersDataTable: React.FC<OrdersDataTableProps> = ({
       ),
       patient: (
         <ConfigurableLink
-          to={`\${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${targetPatientDashboard}`}
+          to={`\${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${
```

Comment:
Is the `\` necessary for this?
```suggestion
          to={`${openmrsSpaBase}/patient/${order.patient?.uuid}/chart/${
```

Reply:
Hi  @pirupius , The '\\' at the start is required to stop the interpolation of the dollar sign, so the url becomes "${openmrsSpaBase}\..." ,and then in the configurable link, "${openmrsSpaBase}" is interpolated to "/openmrs/spa".
Thank you @pirupius